### ISSUE TITLE
Decrease bundle size by optimising the intlTelInput library

### ISSUE THREAD

**Issue by mordeth** [Automattic/woocommerce-payments#4623]
### Description
By introducing the `intlTelInput` library in #4355, the bundle size was raised by 300kb.

### Acceptance criteria
 - Decrease the bundle size.
 - `intlTelInput` fields functionality shouldn't be changed ( validation and formatting ).

### More context
p1660895622633449-slack-C02C00HCJ8J

**Comment by kalessil** [comment]
@mordeth, if it were any related discussions, please add them under the 'More context' section.

**Comment by mordeth** [comment]
Blocked by pdjVC1-13C-p2

**Comment by mordeth** [comment]
Closing in favor of https://github.com/Automattic/woocommerce-payments/pull/5997
